webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -16,30 +16,63 @@ import { Loadable, Loaded, NotLoaded } from 'utils/loadable';
 import { defaultExperimentColumns } from './glide-table/columns';
 import { Error, Loading, NoExperiments, NoMatches } from './glide-table/exceptions';
 import GlideTable, { SCROLL_SET_COUNT_NEEDED } from './glide-table/GlideTable';
+import { isValidSort, Sort, ValidSort } from './glide-table/MultiSortMenu';
 import TableActionBar from './glide-table/TableActionBar';
 import { useGlasbey } from './useGlasbey';
 
 interface Props {
   project: Project;
 }
 
+const makeSortString = (sorts: ValidSort[]): string =>
+  sorts.map((s) => `${s.column}=${s.direction}`).join(',');
+
 export const PAGE_SIZE = 100;
 const F_ExperimentList: React.FC<Props> = ({ project }) => {
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const [page, setPage] = useState(
+  const [page, setPage] = useState(() =>
     isFinite(Number(searchParams.get('page'))) ? Number(searchParams.get('page')) : 0,
   );
-  const [experiments, setExperiments] = useState<Loadable<ExperimentItem>[]>(
+  const [sorts, setSorts] = useState<Sort[]>(() => {
+    const sortString = searchParams.get('sort') || '';
+    if (!sortString) {
+      return [{ column: undefined, direction: undefined }];
```

Comment:
nit: this should probably import `INITIAL_SORTS` from MultiSortMenu

Reply:
Or maybe define the object as `EMPTY_SORT` or something and have them both use it, idk.

Reply:
good idea!